class-auth.php

```diff
@@ -203,8 +203,12 @@ public function generate_token( $user, $return_raw = true ) {
 
 		$alg = $this->get_alg();
 
+		$keyfile = fopen(JWT_PRIVATE_KEY, 'r');
```

Comment:
It would be ideal that the key is stored in memory after being read in so we don't need to read the file each request. But I think this is fine if we are only using it for automation.